lib/bolt/cli.rb

```diff
@@ -207,35 +197,57 @@ def initialize(options)
             @options[:'sudo-password'] = password
           end
         end
-        define('--configfile CONFIG_PATH',
-               'Specify where to load the config file from') do |path|
+
+        separator 'Run context:'
+        define('--tmpdir DIR', 'The directory to upload and execute temporary files on the target') do |tmpdir|
```

Comment:
This might belong in transports since it's not supported by all of them and effects the target unlike the rest of these